package.json

```diff
@@ -21,7 +21,8 @@
       "import": "./lib/esm/index.js",
       "require": "./lib/commonjs/index.js"
     },
-    "./bundle.css": "./lib/bundle.css"
+    "./bundle.css": "./lib/bundle.css",
+    "./bundle-no-preflight.css": "./lib/bundle-no-preflight.css"
```

Comment:
Is it going to be hard to maintain the promise that the non-preflight css bundle looks as expected?

Reply:
The non-preflight bundle definitely looks off, so users will need to supply resets on their end to have the components look as expected. It will be on them to update those resets as needed to keep the styling consistent

Reply:
In terms of maintaining the library though, I'm a little concerned that we'll add  a new component and forget to specify border in this specific way. I wouldn't want to get random techops or github issues in the future and play whack-a-mole with missing border issues. Do you have any ideas for preventing these kinds of issues?

Reply:
ah I see what you're saying. I updated the Tailwind configs for Storybook and the test-site so we'll get an indication if we missed specifying a border-color. with the default border color changed to black, the styling that we see when testing will match what users will see if they use the no-resets bundle with the Tailwind preflight resets. so if we don't want a black border, we'll know to add a border-color when looking at the component in either Storybook or the test-site

Reply:
ah ok cool!

Reply:
lot more elegant than say taking a separate set of snapshots